README.md

```diff
@@ -34,18 +34,27 @@
 ## Initial setup
 
 ```shell
-git clone git@github.com:daritelska-platforma/frontend.git
+git clone https://github.com/podkrepi-bg/frontend
```

Comment:
I changed it because it's easier for new people to onboard with https. For those who prefer ssh, they can always do, assuming they are more tech savvy.
Interesting discussion here: https://stackoverflow.com/questions/11041729/why-does-github-recommend-https-over-ssh